src/redux/categories/categories.js

```diff
@@ -0,0 +1,40 @@
+// Initial category
+const initialCategoryState = [];
+
+// create action contants - This are calleds action types
+const ADD_CATEGORY = 'category/ADD_CATEGORY';
+const STATUS_CHECKED = 'category/STATUS_CHECKED';
+
+// Action Creators for the action constant
+
+// ########## Action to category ###############/
+const addCategory = (category) => ({
+  type: ADD_CATEGORY,
+  payload: {
+    title: category.title,
+  },
+});
+
+const statusCheck = () => ({
+  type: STATUS_CHECKED,
+  payload: {
+    title: 'Under Construction',
+  },
+});
+
```

Comment:
- [ ] Kindly be noticed that you should only create and export **Action Creator**. (`checkStatus` function), there is no need for the `addCategory` function.